apps/web/pages/api/user/avatar.ts

```diff
@@ -64,6 +65,28 @@ async function getIdentityData(req: NextApiRequest) {
       avatar: team?.logo || getPlaceholderAvatar(null, teamname),
     };
   }
+  if (orgSlug) {
+    const org = await prisma.team.findFirst({
+      where: {
+        slug: orgSlug,
+        metadata: {
+          path: ["isOrganization"],
+          equals: true,
+        },
+      },
+      select: {
+        slug: true,
+        logo: true,
+        name: true,
+      },
+    });
+    return {
+      org: org?.slug,
+      name: org?.name,
+      email: null,
+      avatar: org?.logo || getPlaceholderAvatar(null, org?.name),
```

Comment:
```suggestion
      avatar: getPlaceholderAvatar(org?.logo, org?.name),
```

`getPlaceholderAvatar` already takes care of handling nullish org.logo